app/unit-tests/src/org/commcare/android/tests/formentry/IdentityCalloutTests.kt

```diff
@@ -96,6 +99,25 @@ class IdentityCalloutTests {
         assertEquals(guidToConfidenceMap.elementAt(2), "★★★")
     }
 
+    @Test
+    fun testRegistrationWithTemplates() {
```

Comment:
Can we also verify that converting back base 64 encoded templates to original templates using the support lib method indeed matches with original templates as part of this test.